src/edu/stanford/dlss/wowza/SulWowza.java

```diff
@@ -34,8 +34,8 @@
 {
     static String stacksTokenVerificationBaseUrl;
     static String stacksUrlErrorMsg = "rejecting due to invalid stacksURL property (" + stacksTokenVerificationBaseUrl + ")";
-    static final String HONEYBADGER_KEY = "WOWZA_HONEYBADGER_API_KEY";
-    static final String HONEYBADGER_ENVIRONMENT = "WOWZA_ENVIRONMENT";
+    static final String HONEYBADGER_KEY_ENV_VAR = "WOWZA_HONEYBADGER_API_KEY";
```

Comment:
ha, i'm always worried i'm being too verbose in my naming, but agreed, will change.